src/software/ai/navigator/path_planner/path_planner_test.cpp

```diff
@@ -35,90 +35,81 @@ RobotNavigationObstacleFactory robot_navigation_obstacle_factory(
     Util::DynamicParameters->getAIConfig()->getRobotNavigationObstacleFactoryConfig());
 
 std::vector<PlannerTestCase>
-    test_cases =
-        {{.name               = "Empty field straight line",
-          .start              = Point(0, 0),
-          .end                = Point(1, 0),
-          .navigable_area     = Rectangle({-1, -1}, {2, 2}),
-          .obstacles          = {},
-          .should_return_path = true},
+    test_cases = {{.name               = "Empty field straight line",
+                   .start              = Point(0, 0),
+                   .end                = Point(1, 0),
+                   .navigable_area     = Rectangle({-1, -1}, {2, 2}),
+                   .obstacles          = {},
+                   .should_return_path = true},
 
-         {.name           = "Single stationary robot in path",
-          .start          = Point(0, 0),
-          .end            = Point(2, 0),
-          .navigable_area = Rectangle({-2, -2}, {2, 2}),
-          .obstacles      = {robot_navigation_obstacle_factory.createFromRobotPosition(
-              Point({1, 0}))},
-          .should_return_path = true},
+                  {.name               = "Single stationary robot in path",
+                   .start              = Point(0, 0),
+                   .end                = Point(2, 0),
+                   .navigable_area     = Rectangle({-2, -2}, {2.5, 2.5}),
```

Comment:
ThetaStarPathPlanner cuts off one robot radius from the sides of the navigable area so that it doesn't plan paths that smash into the walls, so the slight increase here accounts for that